openmetadata-ui/src/main/resources/ui/public/locales/en-US/Database/Clickhouse.md

```diff
@@ -3,84 +3,89 @@
 In this section, we provide guides and references to use the Clickhouse connector.
 
 # Requirements
-<!-- to be updated -->
-You can find further information on the Kafka connector in the [docs](https://docs.open-metadata.org/connectors/database/clickhouse).
+You can find further information on the Clickhouse connector in the [docs](https://docs.open-metadata.org/connectors/database/clickhouse).
```

Comment:
Can you add more details around the permission needed to extract metadata as well as add additional information for the other workflows and the queries needed to grant the permissions.

You can see more info here https://github.com/open-metadata/OpenMetadata/blob/7f278c67fd5edb369a79697653f0c66ce8a769a7/openmetadata-ui/src/main/resources/ui/public/locales/en-US/Database/Redshift.md